modules/network-observability-dns-tracking.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-dns-tracking_{context}"]
+= Working with DNS tracking 
+Using DNS tracking, you can monitor your network, conduct security analysis, and troubleshoot DNS issues. You can track DNS by editing the `FlowCollector` to the specifications in the following YAML example..
++
+[IMPORTANT]
+====
+CPU and memory usage increases are observed in the eBPF agent when this feature is enabled.
+====
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-dns_{context}"]
+.Configure `FlowCollector` for DNS tracking
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features:
+      - DNSTracking           <1>                       
+      privileged: true        <2>
+----
+<1> You can set the `spec.agent.ebpf.features` parameter list to enable DNS tracking of each network flow in the web console.
+<2> Note that the `spec.agent.ebpf.privileged` specification value must be `true` for packet drop tracking to be enabled.
+
+
+. When you refresh the *Network Traffic* page, there are new choices to enable in the *Overview* and *Traffic Flow* views as well as new filters.
+.. The *Overview* view has new DNS choices in the in *Manage panels* that you can select to see graphical visualizations and DNS metrics.
+.. In the *Traffic Flows* view new choices are available in *Manage columns* to add DNS columns to the table. 
+.. Next you can filter on specific DNS metrics, such as *DNS Id*, *DNS Latency* and *DNS Response Code* and see more information from the side panel.
+.. In *Query Options* verify that *Show duplicates* is selected in order to show duplicated DNS flows.
```

Comment:
```suggestion
.. In *Query Options* verify that *Show duplicates* is selected to show duplicated DNS flows.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

Reply:
I'm actually going to end up removing this step. Thanks anyways @tmalove , sorry!!